README.md

```diff
@@ -626,7 +626,9 @@ ucan-selector = "*" / uri-scheme / ucan-cid
 | `ucan://<did>/*`        | All of any scheme "owned" by a DID            |
 | `ucan://<did>/<scheme>` | All of scheme "owned" by a DID                |
 
-`ucan:*` represents all of the UCANs in the current proofs array. If selecting a particular proof (i.e. not the wildcard), then its CID MUST be used. In the case of selecting a particular proof, the validator MUST check that the delegated content address is listed in the proofs (`prf`) field.
+`ucan:./*` represents all of the UCANs in the current proofs array. If selecting a particular proof (i.e. not the wildcard), then its CID MUST be used (`ucan:<cid>`). In the case of selecting a particular proof, the validator MUST check that the delegated content address is listed in the proofs (`prf`) field.
+
+`ucan:*` is very powerful and deserves special mention. It selects _any_ UCAN that the issuer has access to (including transitively), even if it is not in the proofs of the current UCAN. This is useful when delegating permissions to another agent, including all unknown future delegations to the issuer.
```

Comment:
Much better 👍 So like, for example in our setup, this could be used when doing device linking? You do this `ucan:*` delegation from agent A to agent B, then later give it some UCANs where the audience is agent A, and no further delegations are needed?

Reply:
nice, very good example! maybe we can add it to the prose here.

Reply:
Lemme know what you think: https://github.com/ucan-wg/spec/blob/fix-typo/README.md#411-ucan-example